bgp.go

```diff
@@ -70,6 +70,17 @@ func (b *BGP) ClearPeers() *BGP {
 	return b
 }
 
+// WithName sets name for the peer.
+func (b *BGPPeer) WithName(name string) *BGPPeer {
+	b.pb.Name = name
+	return b
+}
+
+// Name returns the name of the peer.
+func (b *BGPPeer) Name() string {
```

Comment:
I actually don't think this method is necessary, because the code that accesses it can just reach into the protobuf field itself